Tasks/AzurePowerShell/RunAzurePowerShell.ps1

```diff
@@ -20,5 +20,8 @@ Write-Host "ScriptPath= $ScriptPath"
 $scriptCommand = "& `"$ScriptPath`" $scriptArguments"
 Write-Host "scriptCommand= $scriptCommand"
 Invoke-Expression -Command $scriptCommand
+if ($LASTEXITCODE -ne 0) {
+    Write-Error -Message "##[error]BUILD FAILED: $LASTEXITCODE"
```

Comment:
you don't need the ##[error] here we have overridden the Write-Error so it should take care of the formatting for you for the log highlighting.